ignite/pkg/localfs/watcher.go

```diff
@@ -98,9 +99,7 @@ func Watch(ctx context.Context, paths []string, options ...WatcherOption) error
 	w.wt.IgnoreHiddenFiles(w.ignoreHidden)
 
 	// add paths to watch
-	if err := w.addPaths(paths...); err != nil {
-		return err
-	}
+	w.addPaths(paths...)
```

Comment:
Any reason why we don't handle this error anymore?

Reply:
Forgot to add it back apparently. Missed that one. Fixed now.